src/tracksdata/attrs.py

```diff
@@ -75,11 +101,14 @@ class AttrComparison:
         The attribute to compare.
     op : Callable
         The operator to use for the comparison.
-    other : ExprInput
+    other : ExprInput | MembershipExprInput
         The value to compare the attribute to.
     """
 
-    def __init__(self, attr: "Attr", op: Callable, other: ExprInput) -> None:
+    def __init__(self, attr: "Attr", op: Callable, other: ExprInput | MembershipExprInput) -> None:
+        if _is_membership_expr_input(other) and op != _in_op:
+            raise ValueError("Membership values can only be used with the 'in' operator.")
```

Comment:
I think this might be easier for an user to debug because they will be using `Attr.is_in(...)`.

```suggestion
            raise ValueError("Membership values can only be used with the 'is_in' method.")
```